### ISSUE TITLE
Make requesting songs in chat less verbose (option)

### ISSUE THREAD

**Issue by Gol-D-Ace** [Hyphen-ated/DJBot#27]
Having an option to make the chat output less verbose (or maybe turn it completly off) when a user requests a song would be nice.

Example: Your song got added username

**Comment by Hyphen-ated** [comment]
seems like it would be really bad for usability to turn it off completely. users wouldn't know they made a typo like "!songrequst" until the song just never played.
what would you want the less-verbose version to look like? "song added"?

note that you can already turn off the "up next" messages if you just want less overall verbosity

**Comment by LazyR1co** [comment]
I'd like something similar. 

I use OBS Chat by nightdev to display chat on screen stream. Using this though, the chat gets very full of text from the DJbot, as my bot is unable to be invisible in that obs chat. (Doesn't get recognized by Night OBS chat because its name is Lazyb0t, instead of LazyBot.) 

As an alternative to get my bot to not spam the onscreen chat I use "! " at the beginning of most responses that my bot spits out, so that the msgs from my bot won't show up on screen.

Basically I'd like the option of adding "! " to the beginning of everything DJBot says.

I tried to do this myself, and was able to edit DjService.java and find and replace
irc.message(" 
to 
"irc.message("! "

But I have zero real experience coding, much less with java or anything close to it, and I tried for several hours to make the change myself, but wasn't able to make it work. =\

**Comment by Hyphen-ated** [comment]
I can add a "chat prefix" option that will accomplish that. It won't be in a release for a little while though.

If you want the satisfaction of having your thing work, here's how: make sure you have a jdk and the program "maven" installed. Make sure to edit your PATH and JAVA_HOME as described on https://maven.apache.org/download.cgi

Then run the command "mvn clean package assembly:single" in the project folder, where the pom.xml file lives. It will make an install zip in the target/ directory
